.github/workflows/envoy-prechecks.yml

```diff
@@ -24,17 +24,13 @@ permissions:
 jobs:
   env:
     uses: ./.github/workflows/_env.yml
-  cache:
-    needs:
-    - env
-    uses: ./.github/workflows/_cache_docker.yml
     with:
-      image_tag: "${{ needs.env.outputs.build_image_ubuntu }}"
+      prime_build_image: true
+      check_mobile_run: false
```

Comment:
Should `check_mobile_run` be `true`?

Reply:
not for these prechecks - the effect is that it doesnt need to checkout the full-depth repo (as it does for mobile)